File: 23eg112c21_py/Functions/prime_or_not.py
def prime_or_not(n):
    factor = 0
    for i in range(1,n):
        if n%i == 0:
            factor += 1
    
    if factor==1:
        print("The number is prime")
    else:
        print("The number is not prime")

num = int(input("Enter a number: "))
prime_or_not(num)

File: 23eg112c21_java/basic/prime_or_not.java

public class prime_or_not {
    public static void main(String[] args) {
        int n = 25;
        int c = 0;
        int i;
        for (i = 1; i < n; i++) {
            if (n % i == 0) {
                c = c + 1;
            }
        }
        if (c == 1) {
            System.out.println("Prime");

        } else {
            System.out.println("Not Prime");
        }
    }
}